### PR TITLE
cleanup: remove symlinks and use a default path

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area blog

**What this PR does / why we need it**:

This PR applies the weekly image to all weekly blog posts

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: layouts/partials/featured-image.html [hunk 1]
```diff
@@ -9,6 +11,8 @@
     <figcaption class="mt-2 mt-md-0"><small class="text-muted">{{ . | html }}</small></figcaption>
   {{ end }}
   </figure>
+{{ else if strings.Contains $title $falco_weekly_prefix }}
```
Comment (LucaGuerra) [inline layouts/partials/featured-image.html:14]: Since it's a prefix, is it possible to do something like `hasPrefix`?

Comment (Andreagit97) [reply layouts/partials/featured-image.html]: Done!

**Review by LucaGuerra — APPROVED** [review]

**Review by jasondellaluce — APPROVED** [review]

**Pull request merged**